src/nplinker/straincollection.py

```diff
@@ -0,0 +1,142 @@
+import csv
```

Comment:
Rename the filename to `strain_collection.py`? to follow the style of e.g. `molecular_family.py`